python/paddle/fluid/tests/unittests/parallel_executor_test_base.py

```diff
@@ -76,7 +76,7 @@ def run_executor(exe, binary, feed, fetch_list):
                 build_strategy=build_strategy,
                 exec_strategy=exec_strategy)
         else:
-            binary = compiler.CompiledProgram(main)
```

Comment:
In origin implementation, the `binary` is executed by `executor` as a `program`, but now, the `compiled program` is executed by `parallel executor`.